src/include/sof/audio/audio_stream.h

```diff
@@ -73,6 +73,9 @@ struct audio_stream {
 	 */
 	uint16_t frame_align_shift;
 
+	signed char fmt_shift;	/**< Use to calculate samples */
```

Comment:
there is already a frame_shift_align above, can we reuse ? I would also include in the coment to "avoid divisions"